api/go.mod

```diff
@@ -0,0 +1,28 @@
+module github.com/cosmos/ibc-go/api
+
+go 1.22.0
```

Comment:
should we be using go 1.21?

Reply:
I don't think this really matters since its hitting a feat branch for now. I imagine by release time all should be updated to go 1.22

Reply:
Yeah, v9 will be in 1.22.